test/unit/models/requests.test.ts

```diff
@@ -1,71 +1,86 @@
 import { expect } from "chai";
 import cleanDb from "../../utils/cleanDb";
 import { createRequest, updateRequest } from "../../../models/requests";
-import { createOooStatusRequests, updateOooApprovedRequests,updateOooRejectedRequests } from "./../../fixtures/oooRequest/oooRequest";
+import {
+  createOooStatusRequests,
+  updateOooApprovedRequests,
+  updateOooRejectedRequests,
+} from "./../../fixtures/oooRequest/oooRequest";
+import { REQUEST_ALREADY_PENDING } from "../../../constants/requests";
 
 describe("models/oooRequests", () => {
-    afterEach(async () => {
-        await cleanDb();
-    });
+  afterEach(async () => {
+    await cleanDb();
+  });
 
-    describe("createRequest", () => {
-        it("should add a new OOO request to the database", async () => {
-            const oooRequest = await createRequest(createOooStatusRequests);
-            expect(oooRequest).to.not.be.null;
-            expect(oooRequest).to.have.property("id");
-            expect(oooRequest).to.have.property("requestedBy");
-        });
+  describe("createRequest", () => {
+    it("should add a new OOO request to the database", async () => {
```

Comment:
instead of `should add a new OOO request to the database,` We could use a more descriptive name like `should successfully create a new OOO request.`